code/__defines/_planes+layers.dm

```diff
@@ -237,7 +237,7 @@ What is the naming convention for planes or layers?
 	layer = initial(layer)
 
 /*
-  PLANE MASTERS
```

Comment:
this kind of thing is done deliberately for formatting. Whitespace inside comments should not be touched

Reply:
[space] can give error then [tab] in run_linters